docs/docs/adrs/adr-018-permissionless-ics.md

```diff
@@ -0,0 +1,342 @@
+| `ConsumerCommissionRatePrefix`           | Comission rate per chain per validator                                                                                                                                        |**YES**                   | Every `MsgSetConsumerCommissionRate` message                                                             |
+| `MinimumPowerInTopNBytePrefix`           | Stores the minimum power needed to opt in for a chain                                                                                                                         |**YES**                   | Every epoch                                                                                              |
+| `ConsumerAddrsToPruneV2BytePrefix`       | Stores consumer addresses to be pruned (as part of `VSCMaturedPacket`s deprecation)                                                                                           |**YES**                   | Every `MsgAssignConsumerKey` or `MsgOptIn` and later during actual pruning                               |
+
+Everything stored under a key associated with a `chainId` needs to be migrated to new state under `consumerId`.
+
+### New Messages
+In what follows, we describe the new messages (i.e., `MsgRegisterConsumerChain`, `MsgInitiateConsumerChain`, and `MsgUpdateConsumerChain`)
+that Permissionless ICS introduces, and on how those can be used.
+Then, we describe how to utilize these messages with our existing codebase.
+
+#### Register a Consumer Chain
+We first have to register an Opt In chain before launching it. This is done through the following message.
+```protobuf
+message MsgRegisterConsumerChain {
+  // the title of this launch 
+  string title;
+  // the description of this consumer chain
+  string description;
+  // the chain id of the new consumer chain
+  string chain_id;
+  // the owner of this consumer chain
+  string owner_address;
+}
+```
+
+This response of this message contains a single `string`, that is the `consumerId` for this registered consumer chain and initiates
+a consumer chain in its waiting phase. With the returned `consumerId`, validators can already opt in on the consumer
+chain to show their potential interest on the chain. Additionally, a front-end ICS launchpad can also present
+this chain. Additionally, this allows consumer chains to show that they are interested in joining ICS even though,
+they might not yet know the specific ICS parameters (see `MsgInitiateConsumerChain`) they would like to use. 
+
+This message contains the `owner_address` that corresponds to the address that would be able to initiate or later update this consumer chain.
+We store the owner address of each Opt In consumer chain by creating an association between `consumerId`s and `owner_address`es.
+Top N chains do not have an `onwer_address` because they can only be modified through governance proposals. 
```

Comment:
Isn't that the governance account?

Reply:
Yes, but there's no benefit for `owner_address` to contain something when we have Top N chains.